src/test/utils/override-cli-opts.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const Bluebird = require('Bluebird');
+const Bluebird = require('bluebird');
```

Comment:
Not sure how I missed this on the merge, I ran the tests I'm 99% positive... :-/